### DISCUSSION TITLE
ioPay Wallet Reviewing problem

### DISCUSSION THREAD

**Discussion by anshuomomo** [WalletConnect/walletconnect-monorepo#1454] [Developer Support] [answered]
Hi team,

It has been a long time that our ioPay wallet project in PENDING REVIEW. Last reply from your team was ask us to upload example videos to show how ioPay wallet support wallet connect. Could you please have a look and review our app?

Here is the project information:
Our project id: 4202cfb72696dde11177cd3172ea9f49
Example videos with https://example.walletconnect.org/:
Android: https://youtube.com/shorts/NS-bNMhJp60?feature=share
iOS: https://youtu.be/XOhRJ70AE-c?feature=share

Best regards,
Thanks

**Comment by looter** [comment, marked as answer]
Hi @anshuomomo, thanks for your patience. Appreciate you sending the video and the context. Upon further review it looks like there are two ioPay listings:
1. Recently published: https://cloud.walletconnect.com/admin/registry-listing?uuid=a8fe41ca-43f5-4c9a-b114-243a55d3a491
2. Previous:  https://cloud.walletconnect.com/admin/registry-listing?uuid=798edb1d-4dd4-46c5-a057-519f0d8bc441

The second one also uses an iotex.com domain. Would you like me to connect the prior one to the account as well? Let me know!

**Comment by anshuomomo** [comment]
Hi @looter , Such a good news for our app being published. Thank you!  We did have a previous version submmited years ago. You can connect to the account or just deleted the previous version( It's an old version app without multichain supported https://github.com/WalletConnect/walletconnect-registry/issues/214).
